src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -370,7 +370,7 @@ public function each(\Closure $closure)
      */
     public function slice($offset = 0, $length = -1)
     {
-        return new static(iterator_to_array(new \LimitIterator($this, $offset, $length)), $this->uri);
```

Comment:
Yes, in #15934 I asked to backport this to the `2.3` branch, but realised that the `slice()` method was introduced later. That's why I did not backport your changes to Symfony 2.3.